spec/ruby_llm/chat_assume_model_exists_spec.rb

```diff
@@ -78,5 +78,15 @@
       expect(chat.model.id).to eq(custom_model)
       expect(chat.model.provider).to eq(provider)
     end
+
+    it 'works with .with_schema method' do
+      chat = RubyLLM.chat(
+        model: 'gemini-pro',
+        provider: 'gemini',
+        assume_model_exists: true
```

Comment:
I'd rather use the Ollama provider for this one and a randomized name of a model.